src/main/scala/no/ndla/conceptapi/service/search/DraftConceptIndexService.scala

```diff
@@ -46,7 +46,7 @@ trait DraftConceptIndexService {
       mapping(documentType).fields(
         List(
           intField("id"),
-          keywordField("defaultTitle"),
+          keywordField("defaultTitle").fields(keywordField("lower").normalizer("lower")),
```

Comment:
Siden `defaultTitle` allerede er et keyword field og ikke brukes til noe annet enn sortering så kan vi kanskje bare normalize feltet selv?

```suggestion
          keywordField("defaultTitle").normalizer("lower"),
```

Reply:
Godt poeng!

Reply:
Har oppdatert både for drafts og publiserte.